setup-enzyme.js

```diff
@@ -1,5 +1,6 @@
 import { configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
+import '@testing-library/jest-dom/extend-expect.js';
```

Comment:
I currently do not have a preference but we can always whitelist which matchers we want:

```
import {toBeInTheDocument, toHaveClass} from '@testing-library/jest-dom'

expect.extend({toBeInTheDocument, toHaveClass})
```

Reply:
If we did it this way, I'd prefer to include it at a per-test level.